README.md

```diff
@@ -26,7 +26,7 @@ Welcome! To get set up, please follow [the installation guide](docs/01-start-her
 
 [The best place to start is here](docs/01-start-here)
 
-Fixes for common problems can be found [here](docs/01-start-here/03-troubleshooting.md).
+Fixes for common problems can be found [here](docs/01-start-here/04-troubleshooting.md).
```

Comment:
That's kind of the problem with ordering being part of the name. Any manually created link to a file must be updated if new file is added or order changes.
Let's see if it becomes a major problem.
Eventually I would like to revamp the main README as well